test/JDBC/input/functions/datepart-vu-verify.sql

```diff
@@ -210,3 +210,54 @@ GO
 
 SELECT DATEPART(yy, '220101 14:30:00.97531086 +13:14')
 GO
+
+
+-- Test Case for Date Part Functions Timezone Invariance
+DECLARE @current_timezone VARCHAR(50);
+DECLARE timezone_cursor CURSOR FOR SELECT TimezoneName FROM date_part_vu_prepare_TestTimezones;
+
+OPEN timezone_cursor;
+FETCH NEXT FROM timezone_cursor INTO @current_timezone;
+
+WHILE @@FETCH_STATUS = 0
+BEGIN
+    -- Set the timezone
+    EXEC('SELECT set_config(''timezone'', ''' + @current_timezone + ''', false)');
+
+    -- Insert results for this timezone
+    INSERT INTO date_part_vu_prepare_TestResults (TestCase, TimeZone, InputDate, Day, Month, Year)
+    SELECT 
+        'Test: ' + CONVERT(VARCHAR, d.TestDate, 120) + ' in ' + @current_timezone AS TestCase,
+        @current_timezone AS TimeZone,
+        d.TestDate AS InputDate,
+        DAY(d.TestDate) AS [Day],
+        MONTH(d.TestDate) AS [Month],
+        YEAR(d.TestDate) AS [Year]
```

Comment:
Please also add tests for `datepart` and `datename` components listed in TSQL docs.